docs/guides/langchain.md

```diff
@@ -0,0 +1,114 @@
+# Building a langchain AI bot for conversation using Prem-Operator
+
+## Requirements
+
+- Kubernetes deployment with atleast 1 GPU based node, if you are using CPU based node consider using smaller models
+- Prem Operator v1.0.0 or above
```

Comment:
I don't think we should start with v1.0.0, people's expectations start to get wild.

Reply:
Good point, I was also think to make it 0.1 release or something?

Reply:
I'd just stick with 0.10.1